src/ncurses/window.cr

```diff
@@ -3,6 +3,7 @@ require "./raw_window"
 
 module NCurses
   class Window
+    property closed : Bool = false
```

Comment:
What is the purpose of this line? 
I think that `closed` is a private (internal) property or read-only property.
If `closed` property is assigned true by external code by mistake, the window resource will never be closed.

Would you give us any examples to use `closed` property?

I agree to add `closed?` method to read `@closed` state!